Evil.java

```diff
@@ -25,8 +25,10 @@ private void intialiseEvil(int r, int c){
 	}
 	
 	public int[] suggestMove(){
+		int direction = upOrDown();
 		int [] location = {rowCoord,colCoord};
 		location[1] = colCoord-1;
```

Comment:
http://programmers.stackexchange.com/questions/2783/space-around-operators-a-b-2-or-a-b-2-what-are-the-arguments - Really helps make the code clearer to have whitespace around operators